xapian-applications/omega/docs/overview.rst

```diff
@@ -307,8 +307,8 @@ other filters too - see below):
 * Atom feeds (.atom)
 * MAFF (.maff) if unzip is available
 * MHTML (.mhtml, .mht) if perl with MIME::Tools is available
-* MIME email messages (.eml) and USENET articles if perl with MIME::Tools and
-  HTML::Parser is available
+* MIME email messages (.eml) and USENET articles if gmime 2.6 or perl with MIME::Tools and
+  HTML::Parser are available
```

Comment:
I think this should still be "is" - it's "... if A or B is available" where A is "gmime 2.6" and B is "perl with <module list>".